apps/mobile/lib/modules/core/core_module.dart

```diff
@@ -10,5 +12,6 @@ class CoreModule extends Module {
           export: true,
         ),
         Bind.lazySingleton((i) => NotificationService(), export: true),
+        Bind.lazySingleton((i) => ApiClient(), export: true),
```

Comment:
Use the static  method to instance the api client

Reply:
Resolved.